tasks/skylab/LoomUtils.wdl

```diff
@@ -3,7 +3,7 @@ version 1.0
 task SmartSeq2LoomOutput {
   input {
     #runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-loom-output:0.0.4-ss2-loom-fix-1"
+    String docker = "quay.io/humancellatlas/secondary-analysis-loom-output:0.0.5-fk"
```

Comment:
why did this change? None of the changes above are included in this docker image

Reply:
discussed